src/components/App.tsx

```diff
@@ -26,7 +26,7 @@ function App() {
     if (!signerNetwork) return
 
     setSwitchNetworkModalVisible(signerNetwork !== networkName)
-  }, [setSwitchNetworkModalVisible, signerNetwork])
+  }, [networkName, signerNetwork])
```

Comment:
setState methods are stable and don't need to be included in deps

![image](https://user-images.githubusercontent.com/238271/142787445-f6c727ff-984e-487d-81a4-04c8130abe9b.png)
([source](https://reactjs.org/docs/hooks-reference.html))

Reply:
awesome. pretty sure this needs correcting in a lot of places

Reply:
I didn't catch them all but I think I got a bunch of them.